bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v19/BitcoindV19RpcClient.scala

```diff
@@ -0,0 +1,77 @@
+package org.bitcoins.rpc.client.v19
+
+import akka.actor.ActorSystem
+import org.bitcoins.core.currency.Bitcoins
+import org.bitcoins.rpc.client.common.{BitcoindRpcClient, BitcoindVersion}
+import org.bitcoins.rpc.config.BitcoindInstance
+import org.bitcoins.rpc.jsonmodels.{GetBalancesResult, SetWalletFlagResult}
+import play.api.libs.json.Json
+import play.api.libs.json.JsString
+import org.bitcoins.rpc.serializers.JsonSerializers._
+
+import scala.concurrent.Future
+import scala.util.Try
+
+/**
+  * Class for creating a BitcoindV19 instance that can access RPCs
+  * @param instance
+  * @param actorSystem
+  */
+class BitcoindV19RpcClient(override val instance: BitcoindInstance)(
+    implicit
+    actorSystem: ActorSystem)
+    extends BitcoindRpcClient(instance)
+    with V19BlockFilterRpc {
+
+  override lazy val version: BitcoindVersion = BitcoindVersion.V19
+
+  /**
+    * setWalletFlag
+    *
+    * Change the state of the given wallet flag for a wallet.
+    */
+  def setWalletFlag(
+      flag: String,
```

Comment:
There is only one currently `avoid_reuse`

Reply:
Gotchya, well in any case I'd say it would be nice to have an ADT (Abstract Data Type) for this.

This will entail creating a
```sealed abstract class WalletFlag```
with just the one member for now, and usually for readability with things like this we put the instances inside a companion object:
```
object WalletFlag {
  case object AvoidReuse extends WalletFlag {
    override def toString: String = "avoid_reuse"
  }
}
```
Using the keyword `sealed` enforces that this `abstract class` can only be `extend`ed from within the same file. This gives us nice features like if you ever add a `WalletFlag`, you will get compiler warnings about all of the now non-exhaustive `match` statements where the new case needs to be handled.